VTCareService/src/main/java/edu/vt/cs/vtcare/vtcareservice/api/VTCareResource.java

```diff
@@ -301,4 +301,17 @@ public Response rescheduleAppointment(@PathParam("appointment-id")
                                 "appointment.");
         }
     }
+
+    @GET
+    @Path("patients/{patient-id}/getHistory")
```

Comment:
since its already a GET call, just writing "history" instead of "gethistory" is sufficient.